src/components/DropZone/tests/DropZone.test.tsx

```diff
@@ -80,47 +77,41 @@ describe('<DropZone />', () => {
     const dropZone = mountWithAppProvider(
       <DropZone onDrop={spy} accept="image/jpeg" />,
     );
-    const event = createEvent('drop', files);
-    dropZone.getDOMNode().dispatchEvent(event);
+    fireEvent({element: dropZone});
     expect(spy).toHaveBeenCalledWith(files, acceptedFiles, rejectedFiles);
   });
 
   it('calls the onDropAccepted callback with acceptedFiles when it accepts only jpeg', () => {
     const dropZone = mountWithAppProvider(
       <DropZone onDropAccepted={spy} accept="image/jpeg" />,
     );
-    const event = createEvent('drop', files);
-    dropZone.getDOMNode().dispatchEvent(event);
+    fireEvent({element: dropZone});
     expect(spy).toHaveBeenCalledWith(acceptedFiles);
   });
 
   it('calls the onDropRejected callback with rejectedFiles when it accepts only jpeg', () => {
     const dropZone = mountWithAppProvider(
       <DropZone onDropRejected={spy} accept="image/jpeg" />,
     );
-    const event = createEvent('drop', files);
-    dropZone.getDOMNode().dispatchEvent(event);
+    fireEvent({element: dropZone});
     expect(spy).toHaveBeenCalledWith(rejectedFiles);
   });
 
-  it('calls the onDragEnter callback when a dragEnter event is fired', () => {
+  it('calls the onDragEnter callback when a dragenter event is fired', () => {
     const dropZone = mountWithAppProvider(<DropZone onDragEnter={spy} />);
-    const event = createEvent('dragenter', files);
-    dropZone.getDOMNode().dispatchEvent(event);
+    fireEvent({element: dropZone, eventType: 'dragenter'});
     expect(spy).toHaveBeenCalled();
   });
 
   it('calls the onDragOver callback when a dragOver event is fired', () => {
```

Comment:
If we're fixing the typo above as might as well do it here as well 😄 

```suggestion
  it('calls the onDragOver callback when a dragover event is fired', () => {
```